source/concepts/applications.rst

```diff
@@ -231,52 +176,106 @@ The command does not return anything. This leads to the following signature:
-:math:`\mathbf{C}_i` for avatar :math:`i` is computed using polar coordinates converted
-to Cartesian. Each avatar is assigned an angle :math:`\theta_i`:
+players, and :math:`r` the radius given as an argument. Then the center's position
+:math:`\mathbf{C}_p` for avatar :math:`p` is computed using polar coordinates converted
+to Cartesian. Each avatar is assigned an angle :math:`\theta_p`:
 
 .. math::
 
-  \theta_i = \frac{i \times 2 \pi}{N}
+  \theta_p = \frac{ 2 \pi p}{N}
 
-Then the positions is:
+Then each position is:
 
 .. math::
 
   \begin{align}
-  \mathbf{C}_i &= \begin{bmatrix}
-    r\cos{\theta_i}\\
+  \mathbf{C}_p &= \begin{bmatrix}
+    r\cos{\theta_p}\\
     0\\
-    r\sin{\theta_i}\\
+    r\sin{\theta_p}\\
   \end{bmatrix}
   \end{align}
 
-The rotation :math:`\mathbf{R}_i` is expressed as a quaternion and is defined as:
+The rotation :math:`\mathbf{R}_p` is expressed as a quaternion and is defined as:
 
 .. math::
 
    \begin{align}
-   \mathbf{R}_i &= \begin{bmatrix}
+   \mathbf{R}_p &= \begin{bmatrix}
      0\\
-     \sin{\frac{1}{2} \big(-\theta_i - \frac{2\pi}{N}\big)}\\
+     \sin{\frac{1}{2} \big(-\theta_p - \frac{2\pi}{N}\big)}\\
      0\\
-     \cos{\frac{1}{2} \big(-\theta_i - \frac{2\pi}{N}\big)}\\
+     \cos{\frac{1}{2} \big(-\theta_p - \frac{2\pi}{N}\big)}\\
     \end{bmatrix}
    \end{align}
 
+.. _user-origin-description:
+
+User origin
+~~~~~~~~~~~
+
+A user-origin is a suggestion to the client of how to position its coordinate
+space (and therefore avatar) relative to server space. This is used by the
+:ref:`radial orient <radial-orient>` server feature.
+
+.. note::
+
+   Any client can add user-origin keys. This can be used, for instance, to
+   prototype alternatives to the radial orient feature without modifying the server.
+
+The user origin describes where the server suggests a given user places the center
+of its client space and how to orient it. The origin is described as a protobuf
+Struct under the key ``user-origin.<PLAYER_ID>`` where ``<PLAYER_ID>`` is the ID
+of the user to whom the suggestion is addressed. The Struct has the following keys:
+
+* ``position`` is the suggested location of the center for the user's client
+  space in the server space;
+* ``rotation`` is a quaternion describing the rotation of the user's client
+  space in the server space.
+
+Clients are free to ignore the user-origin suggestion and locate themselves in
+the server space as they choose.
+
+.. warning::
+
+   Any client can add user-origin keys. If used without due care and
+   responsibility a user in VR could get very nauseous.
+
+As a summary, the user origin is specified as follows in the shared state:
+
+.. code::
+
+   user-origin.<PLAYER_ID>: {
+     position,
+     rotation,
+   }
+
+
+.. _multiplayer-update-index:
+
+Update index
+~~~~~~~~~~~~
+
+If the client needs more precise knowledge of which of its updates have already
+been received and rebroadcast to all clients, it can choose to maintain an
```

Comment:
Why is this "rebroadcast?" Does it mean broadcast by the server and received by the clients? In which case, should it be "broadcast and received to all clients".

Reply:
I have changed it to "accepted by the server and broadcast to all clients" -- it would be somewhat misleading to say that you can tell if other clients have received it or not